tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml

```diff
@@ -1,3 +1,29 @@
+trigger:
+  branches:
+    include:
+    - main
+    - rel-*
+  paths:
+    exclude:
+    - docs/**
+    - README.md
+    - CONTRIBUTING.md
+    - BUILD.md
+    - 'js'
```

Comment:
Some mobile pipelines also use the js folder. But I am not sure who they are. 
Training also uses js.